client/src/components/ProTx/components/charts/predictiveFeaturesTableData.js

```diff
@@ -0,0 +1,69 @@
+export const PREDICTIVE_FEATURES_TABLE_DATA = [
+  {
+    Demographic_Feature: 'number of single parents',
```

Comment:
Are these label changes (i.e. all lower case) to match document?

Reply:
D'oh! Good catch. I am going to tweak the text to use Capitals as appropriate then push that into this commit before merging.